tendrl/ceph_integration/manager/manager.py

```diff
@@ -204,7 +157,7 @@ def main():
                 cluster_id = sys.argv[2]
                 utils.set_tendrl_context(cluster_id)
```

Comment:
https://github.com/Tendrl/node_agent/issues/116 introduces starting of integration processes as services and not as command line process.